concepticondata/conceptlists.tsv

```diff
@@ -311,4 +311,4 @@ Gauchat-1925-480	Gauchat, Louis and Jeanjaquet, Jules and Tappolet, Ernest	1925
 Ratcliffe-2020-100	Ratcliffe, Robert R.	2020		100	areal, questionnaire	English	Arabic dialects	https://doi.org/10.1163/22105832-01001100	Ratcliffe2020		This concept list is supposed to reflect [Swadesh's list of 100 items](:ref:Swadesh-1955-100) and was used for a lexicostatistic study of Arabic dialects. The mappings may differ in some cases from the original list of Swadesh, as we had problems to identify some meanings unambiguously from the glosses (for RAIN, for example, no further specification was given). The list was provided in WORD format in the supplementary material of the study and turned out to be corrupted by space characters inserted inside almost all words including the numbers. These spaces were deliberately deleted in our version, to allow for a convenient inspection of the list.	1-29	
 Davletshin-2012-100	Davletshin, Albert	2012		100	basic, proto-language	English	Uto-Aztecan languages	https://doi.org/10.31826/9781463235406-005	Davletshin2012	Davletshin2012	The concepts for this list were taken from [Swadesh's 100-item list](:ref:Swadesh-1955-100) and formed the basis for comparing 9 different Uto-Aztecan languages. Based on this comparison, the author proposed possible corresponding forms of Proto-Aztecan. The concept “small” is actually missing from the proto-word list but was included in the Aztecan word list on pp. 84-90.	91	
 Yao-2017-1100	Yao, Zhao and Wu, Jia and Zhang, Yanyan and Wang, Zhenhong	2017		1100	ranked	English	Chinese	https://link.springer.com/article/10.3758/s13428-016-0793-2#Sec12	Yao2017		This data set contains ratings for affective variables (i.e., valence and arousal) and various semantic variables (concreteness, imageability, context availability, and familiarity) of 1,100 Chinese words. The ratings for all variables were collected with 9-point Likert scales. 	1374-1385	
-Hartmann-2013-162	Hartmann, Iren and Haspelmath, Martin and Taylor, Bradley	2013		162	questionnaire	English	Global	http://valpal.info	Hartmann2013		The online database ValPaL (Valency Patterns Leipzig) was created to study behaviour of verbs in different valency contexts. The concepts in this list are meant to be representative of the verbal lexicon and are reported as showing distinctive syntactic behaviour within and across languages.		
```

Comment:
Nobody else of the more code-savvy people wrote anything about it so I'm sure there's something I'm missing here, but this looks like another entry (Hartmann) was deleted? Same goes for Hartmann in the references.bib file, is that perhaps why the checks fail? (If not, please just ignore this comment.)

Reply:
Yes. I didn't get what to do about this. What should be done from me?